Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -271,3 +309,58 @@ void MenuBar::InstallWAD()
 
   result_dialog.exec();
 }
+
+void MenuBar::PerformOnlineUpdate(const std::string& region)
+{
```

Comment:
I'm not sure where else to place this function... Add this function as a slot on MainWindow, like how it's done for state loads? (I'm not really familiar with Qt concepts or the organisation.)